static/app/views/issueDetails/groupActivity.spec.jsx

```diff
@@ -338,4 +340,63 @@ describe('GroupActivity', function () {
       expect(deleteMock).toHaveBeenCalledTimes(1);
     });
   });
+
+  it('renders ignored', function () {
```

Comment:
```suggestion
  it("renders regular ignored message if org doesn't have `escalating-issues-ui` feature", function () {
```

Should there also be a test for `it("renders regular ignored message if ignoreUntilEscalating is false")`?